src/Core/View.js

```diff
@@ -169,6 +171,7 @@ class View extends THREE.EventDispatcher {
         super();
 
         this.domElement = viewerDiv;
+        this.id = id++;
```

Comment:
I think it should be exposed in the documentation, WDYT @AnthonyGlt @Desplandis ?